package.json

```diff
@@ -39,21 +39,19 @@
   "devDependencies": {
     "@babel/core": "^7.4.3",
     "@babel/runtime": "^7.4.3",
-    "babel-eslint": "^6.1.2",
+    "@react-native-community/eslint-config": "0.0.5",
+    "babel-eslint": "^10.0.1",
     "babel-plugin-module-resolver": "^3.2.0",
-    "babel-preset-airbnb": "^1.1.1",
-    "babel-preset-react-native": "1.9.0",
-    "eslint": "^3.3.1",
-    "eslint-config-airbnb": "^10.0.1",
-    "eslint-plugin-import": "^1.14.0",
-    "eslint-plugin-jsx-a11y": "^2.1.0",
-    "eslint-plugin-prefer-object-spread": "^1.1.0",
-    "eslint-plugin-react": "^6.1.2",
+    "babel-preset-react-native": "4.0.1",
```

Comment:
I think you don't need this directly anymore, nor most of the other plugins as they are not used in the eslint config